Tasks/AzureCLIV2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vsts-tasks-azurecli",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
What does this do?

Reply:
reverting this as package.json has not real impact on the task.